chunk/chunk_store.go

```diff
@@ -206,6 +208,10 @@ type AWSStore struct {
 	tableName  string
 	bucketName string
 
+	// Around which time to start bucketing indexes by day instead of by hour.
+	// Only the day matters, not the time within the day.
+	dailyBucketsFrom model.Time
```

Comment:
Understood. I had to dig into the model code to make sense of that. It's just an alias to a number of milliseconds, rather than, say, a struct. I guess we could make our own type alias, `Day`, to make it clearer, but not worth it IMO.